File: internal/pkg/database/models/metadata_models_defaults.go
package models

import (
	"github.com/gofrs/uuid/v5"
)

type MetadataModelsDefaults struct {
	ID               []string    `json:"id,omitempty"`
	Description      []string    `json:"description,omitempty"`
	MetadataModelsID []uuid.UUID `json:"metadata_models_id,omitempty"`
}

type metadataModelsDefaultsTable struct {
	TableName string

	ID               string
	Description      string
	MetadataModelsID string
}

func MetadataModelsDefaultsTable() metadataModelsDefaultsTable {
	return metadataModelsDefaultsTable{
		TableName: "metadata_models_defaults",

		ID:               "id",
		Description:      "description",
		MetadataModelsID: "metadata_models_id",
	}
}

File: web/src/lib/database/metadata_models_defaults.ts
namespace _MetadataModelsDefaults {
	export enum Column {
		ID = 'id',
		Description = 'description',
		MetadataModelID = 'metadata_models_id'
	}

	export const TableName = 'metadata_models_defaults'

	export interface Interface {
		id?: string[]
		description?: string[]
		metadata_models_id?: string[]
	}
}

export default _MetadataModelsDefaults
